x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -252,6 +253,7 @@ export class TaskStore {
       )
     );
 
+    const trans = apm.startTransaction(`taskManager markAvailableTasksAsClaimed`, 'taskManager');
```

Comment:
I think for this call, we should look into using [`transaction.setLabel()`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/transaction-api.html#transaction-set-label) to add the queue size.  Hopefully it will show up somewhere in the UI, but ideally I'd like some sort of specialzed ui for this anyway, so while it may not be immediately useful, at least it's getting written somewhere.

Reply:
label values must be strings, so not as useful as I thought

Reply:
also, we'd need to actually get the queue size, but I wouldn't want the cost of getting that to be included here, so we'll have to figure out some other story for the queue size ...